tools/topology/topology2/platform/intel/dmic-generic.conf

```diff
@@ -142,6 +142,16 @@ IncludeByKey.PASSTHROUGH {
 					Object.Control.mixer.1 {
 						name '$DMIC0_PCM_NAME Capture Volume'
 					}
+					IncludeByKey.DMIC_MUTE_LED {
+					"true"	{
+							Object.Control.mixer.2 {
+								name '$DMIC0_PCM_NAME Capture Switch'
+
+								mute_led_use 		1
+								mute_led_direction	1
+							}
+						}
+					}
```

Comment:
I don't see the point of making this conditional. We absolutely need a mute control for the DMIC. Whether or not this is connected to LEDs is a separate story.